src/auth/auth.service.ts

```
@@ -17,6 +18,28 @@ import { ConfigService } from '@nestjs/config';
 
 @Injectable()
 export class AuthService {
+  /**
+   * Promote a user to admin. Only super admins can perform this action.
+   * @param requesterId - ID of the user making the request
+   * @param targetUserId - ID of the user to be promoted
+   */
+  async promoteToAdmin(requesterId: string, targetUserId: string): Promise<User> {
```

ESLint

[error] Replace `requesterId:·string,·targetUserId:·string` with `⏎····requesterId:·string,⏎····targetUserId:·string,⏎··`

(prettier/prettier)